bench/AElf.Benchmark/MinerTests.cs

```diff
@@ -6,11 +6,11 @@
 using AElf.Kernel.Blockchain.Application;
 using AElf.Kernel.Blockchain.Domain;
 using AElf.Kernel.Blockchain.Events;
-using AElf.Kernel.EventMessages;
 using AElf.Kernel.Infrastructure;
 using AElf.Kernel.Miner.Application;
 using AElf.Kernel.SmartContract.Infrastructure;
 using AElf.Kernel.SmartContractExecution.Application;
+using AElf.Kernel.SmartContractExecution.Events;

```

Comment:
move to AElf.Kernel.SmartContractExecution.Application,

service and event should put in the same namespace